service/FactomService.js

```diff
@@ -43,14 +29,22 @@ exports.deactivate = function (body) {
 exports.register = function (body) {
     return new Promise(function (resolve, reject) {
         const { jobId: entryHash, options } = body;
+        const network = options.network || Networks.MAIN;
+        if (network !== Networks.MAIN || network !== Networks.TEST) {
```

Comment:
Look carefully at this expression ;)

Why does it currently work and how should it be written to be more logical?

Reply:
Ooof, yeah this would never work. Updated.

Reply:
Yeah okay now, although I would find the DeMorgan counterpart "not(supported1 or supported)" more logical ;)